be/src/storage/rowset/segment.cpp

```diff
@@ -295,6 +295,18 @@ StatusOr<ChunkIteratorPtr> Segment::new_iterator(const Schema& schema, const Seg
     return _new_iterator(schema, read_options);
 }
 
+Status Segment::new_inverted_index_iterator(uint32_t cid, InvertedIndexIterator** iter,
+                                            const SegmentReadOptions& opts) {
+    if (_column_readers[cid] != nullptr) {
```

Comment:
Use _column_readers.find and iterator to check. Do not use operator[] because operator[] will add a new element in _column_readers even _column_readers does not contain cid. It will cause BE crash after fast schema evolution.